assets/runtime/config/gitlabhq/gitlab.yml

```diff
@@ -512,9 +512,9 @@ production: &base
         aws_secret_access_key: '{{AWS_BACKUP_SECRET_ACCESS_KEY}}'
       # The remote 'directory' to store your backups. For S3, this would be the bucket name.
       remote_directory: '{{AWS_BACKUP_BUCKET}}'
-    #   # Use multipart uploads when file size reaches 100MB, see
-    #   #  http://docs.aws.amazon.com/AmazonS3/latest/dev/uploadobjusingmpu.html
-    #   multipart_chunk_size: 104857600
+      # Use multipart uploads when file size reaches 100MB, see
+      #  http://docs.aws.amazon.com/AmazonS3/latest/dev/uploadobjusingmpu.html
+      multipart_chunk_size: {{AWS_BACKUP_MULTIPART_CHUNK_SIZE}}
```

Comment:
It would be neat and for backwards compability to remove the  `multipart_chunk_size` when `AWS_BACKUP_MULTIPART_CHUNK_SIZE ` is not set. I think not everyone needs this directly.